aci/resource_aci_snmpuserp.go

```diff
@@ -0,0 +1,271 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAciSnmpUserProfile() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciSnmpUserProfileCreate,
+		UpdateContext: resourceAciSnmpUserProfileUpdate,
+		ReadContext:   resourceAciSnmpUserProfileRead,
+		DeleteContext: resourceAciSnmpUserProfileDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciSnmpUserProfileImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
```

Comment:
It would be good to check with Lionel on the following: 
1. Is ForceNew:  true required for the attributes: "authorization_key",  "authorization_type", "privacy_key" and "privacy_type"
2. If the above is yes, do we need an update function for this resource?

Reply:
Just checked the comments from Sabari and Lionel. Please ignore the above.